.travis.yml

```diff
@@ -22,6 +27,7 @@ notifications:
 
 before_script:
   - "flake8 ."
+  - if [ $GUI ]; then export DISPLAY=:99.0 && sh -e /etc/init.d/xvfb start -screen 0 1920x1080x24 && sleep 3; fi
```

Comment:
This just needs to be `sh -e /etc/init.d/xvfb start` and the `DISPLAY` can go as a global environment variable.